.github/workflows/test.yml

```diff
@@ -65,6 +65,8 @@ jobs:
               - products/cbbr/**
             checkbook:
               - products/checkbook/**
+            facilities:
+              - products/facilities/**
```

Comment:
and `dcpy/**`!

Reply:
The other option to what I did [here](https://github.com/NYCPlanning/data-engineering/pull/276/commits/4e212c67b28570f752fc955c5b8eb008da571f66) would be just mypying on all (or most of) the codebase separate from triggering product-specific tests.

Reply:
@fvankrieken Ah, I see these have been expanded since my last rebase! And good callout... I'll copy the existing convention for now, and maybe down the line we can extract a `mypy` job that always runs against `dcpy` and certain products. My only concern would be that `mypy` is quite slow... but I'm not entirely sure how slow. Maybe we could speed it up via actions caching with mypy. Looks like there are a few published actions that will cache things.